abp168/shadows_of_knight_ep1_(sam1123_Comments).py

```diff
@@ -0,0 +1,109 @@
+# **********************************************
```

Comment:
Same -- combine with `shadows_of_knight_ep1.py` then delete.

Reply:
Combined and deleted